server/handler.go

```diff
@@ -77,6 +77,10 @@ func (h *Handler) ComQuery(
 ) (err error) {
 	ctx := h.sm.NewContextWithQuery(c, query)
 
+	if strings.ToLower(query) == "show schemas" {
```

Comment:
not related to this PR , but whatever if we can handle `show schemas`, but afaik it's not merged into vitess, yet.

Reply:
Was upload by error sorry